<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VBDEditPage</class>
 <widget class="QWidget" name="VBDEditPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Mode and Position</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="modeLabel">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="modeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="positionComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="priorityGroup">
     <property name="title">
      <string>Disk access priority</string>
     </property>
     <layout class="QVBoxLayout" name="priorityLayout">
      <item>
       <widget class="QSlider" name="prioritySlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="priorityLabelsLayout">
        <item>
         <widget class="QLabel" name="priorityLowLabel">
          <property name="text">
           <string>Lowest</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="prioritySpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="priorityHighLabel">
          <property name="text">
           <string>Highest</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(200, 60, 60); }</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
